packages/amplication-client/src/Project/ProjectSideBar.scss

```diff
@@ -0,0 +1,17 @@
+@import "../style/index.scss";
+
+.project-sidebar {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: var(--large-spacing);
+  gap: var(--large-spacing);
+  width: 280px;
+  background: var(--surface);
+  border: 1px solid var(--black10);
```

Comment:
this should be set on the parent as part of the layout

Reply:
The parent has a different background color. This sidebar is darker than its parent, and it has its own border and border-radius
![image](https://user-images.githubusercontent.com/39680385/179402342-7306efc6-8cbf-49d9-b1f8-c0600f93873a.png)

Reply:
That should be the parent that sets that for all types of content that will be displayed in it - not just on this page but on all page